content/api/commands/mount.md

```diff
@@ -0,0 +1,226 @@
+---
+title: mount
+componentSpecific: true
+---
+
+<Alert type="warning">
+
+Cypress does not have a **cy.mount()** command out-of-the-box. See below for
+info on how to craft your own.
+
+</Alert>
+
+We recommend creating a custom `cy.mount` command which wraps the mount command
```

Comment:
It might be helpful to start this sentence with something "For Component Testing, " with a link to https://deploy-preview-4234--cypress-docs.netlify.app/guides/overview/choosing-testing-type#What-is-Component-Testing